### DISCUSSION TITLE
While building the code I am getting issues

### DISCUSSION THREAD

**Discussion by ShettyPrashanth** [eclipse-cdt/cdt#904] [General]
I am using eclipse IDE(2024-09) for c/c++ developers. I have installed esp IDF plugin and taken a built-example from it. When I try to build it throws some error.

The error is - Errors occurred during the build.
Errors running builder 'CDT Core Builder' on project 'hello_world'.
org/eclipse/cdt/lsp/LspUtils

What is the solution for this problem?

**Comment by jonahgraham** [comment]
Can you reproduce the problem without esp IDF plugin?

**Reply by DrStone1971** [reply]
Hi, 
i have same problem WITH IDF Plugin. I have try to compile  "Hello World" without IDF Plugin, without Any Error and same simple Hello_World using ESP-IDF esp32s3 with error (in compile terminal without error) 

Is it possible that there is an incompatibility between eclipse 2024-09 and the ESP-IDF Plugins for Eclipse 3.0.0.202406051940 plugin?

in attach there is my configuration present on Ubuntu machine, compile log of ESP-IDF Hello_World and error and Compile Standard Hello_World without error.


[error_compile.txt](https://github.com/user-attachments/files/17017540/error_compile.txt)
[config.txt](https://github.com/user-attachments/files/17017542/config.txt)
[compile_std_hello_world.txt](https://github.com/user-attachments/files/17017545/compile_std_hello_world.txt)
[compile_esp_idf.txt](https://github.com/user-attachments/files/17017546/compile_esp_idf.txt)


[config.txt](https://github.com/user-attachments/files/17017455/config.txt)

**Reply by DrStone1971** [reply]
i try to reinstall eclipse 2024-06 with same error on compile.

**Reply by DrStone1971** [reply]
next test
eclipse 2024-09 and ESP-IDF Plugin refferred at https://dl.espressif.com/dl/idf-eclipse-plugin/updates/nightly/ works fine With-out compile problem.

**Reply by jonahgraham** [reply]
It may be that there is such an incompatibility, however the Eclipse CDT project can't really support you on such problems, so I recommend reaching out to the ESP-IDF team for support.

**Reply by DrStone1971** [reply]
it's correct ! Thanks

**Comment by jld01** [comment]
See also: https://github.com/espressif/idf-eclipse-plugin/issues/1047

**Reply by DrStone1971** [reply]
It works ! (Eclipse 2024-09 on Ubuntu and ESP-IDF 5.3.1)

Thanks !!!

DrStone71
